iree/compiler/Codegen/SPIRV/KernelConfig.cpp

```diff
@@ -534,6 +534,8 @@ static LogicalResult setSPIRVOpConfig(const spirv::TargetEnv &targetEnv,
     case spirv::Vendor::AMD:
       result = detail::setAMDCodeGenConfig(targetEnv, rootOp);
       break;
+    case spirv::Vendor::Apple:
+      result = detail::setAppleCodeGenConfig(targetEnv, rootOp);
```

Comment:
`break` is missing here. This will fall to the Mail configuration later.

Reply:
oops, sorry yeah, good catch :)